modules/upload/client-react/containers/Upload.jsx

```diff
@@ -81,13 +98,18 @@ export default compose(
                 __typename: 'File'
               }
             },
-            update: store => {
-              const cachedFiles = store.readQuery({ query: FILES_QUERY });
+            update: ({ caches }) => {
```

Comment:
The destructuring assignment here `{ caches }` tries to access private field of a class, `caches` is a private field. This is forbidden. We must not use private fields. Use public API of the class instead